cmd/containerd/builtins_linux.go

```diff
@@ -4,4 +4,5 @@ import (
 	_ "github.com/containerd/containerd/linux"
 	_ "github.com/containerd/containerd/metrics/cgroups"
 	_ "github.com/containerd/containerd/snapshots/overlay"
+	_ "github.com/containerd/zfs"
 )
```

Comment:
I think we want to support `no_zfs` buildtag

Reply:
Maybe a `omit_default_plugins` tag instead of a tag for each default?

To build with a custom list of plugins someone could add a `builtins_custom.go` with the exact list they want.

Reply:
Your welcome to submit that as a follow up.